opensrp-chw/src/main/java/org/smartregister/chw/dao/WashCheckDao.java

```diff
@@ -19,10 +19,10 @@
 public class WashCheckDao extends AbstractDao {
 
     public static long getLastWashCheckDate(String familyBaseEntityID) {
-        String sql = "select eventDate from event where eventType = 'WASH check' and " +
-                "baseEntityId = '" + familyBaseEntityID + "' order by eventDate desc limit 1";
+        String sql = "select created_at from visits where visit_type = 'WASH check' and " +
```

Comment:
IF using the `created_at` date, you should take the minimum between the `created_at` and `visit_date`. Data synced from the server are assigned  `new Date()` as the default date